packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -257,6 +257,25 @@ export function isCompatVersionBelowMinVersion(minVersion: string, config: Compa
 	return semver.compare(compatVersion, minReqVersion) < 0;
 }
 
+/**
+ * Returns true if the given compat config is compliant with ODSP's version requirements.
+ * ! If a summarizer's version is too old, ODSP will nack the summaries with "Upgrade to a newer version of the Fluid client packages to summarize".
+ */
+export function isOdspCompatCompliant(config: CompatConfig): boolean {
```

Comment:
i don't really get why this is odsp specific, why would we ever test versions older than lts? maybe @scottn12  knows more.

At a minimum more comments that describe why this works, and why we expect it to continue to work would be good.

Reply:
i would kind of expect there to be some constant like `odspMinVersion` or something, but i guess it could be like latest minor per major or something, just not super clear from just the changes, and i don't have context outside of that.

Reply:
> i don't really get why this is odsp specific, why would we ever test versions older than lts? maybe @scottn12 knows more.

Correct me if I'm wrong, but from the PR description it seems like we aren't testing against verisons older than LTS, but rather ODSP does not work with LTS at all. If that's the case I think these changes make sense overall, but I agree with Tony that it would be helpful to add some type of `odspMinVersion` value and add comments to clarify.